spec/system/casa_admins/edit_spec.rb

```diff
@@ -6,27 +6,55 @@
   before { sign_in admin }
 
   context "with valid data" do
-    it "can successfully edit user email and display name" do
-      expected_email = "root@casa.com"
+    it "can successfully edit user display name and phone number" do
       expected_display_name = "Root Admin"
       expected_phone_number = "+14398761234"
 
       visit edit_casa_admin_path(admin)
 
-      fill_in "Email", with: expected_email
       fill_in "Display name", with: expected_display_name
       fill_in "Phone number", with: expected_phone_number
 
       click_on "Submit"
 
       admin.reload
 
-      expect(page).to have_text "New admin created successfully"
-      expect(admin.email).to eq expected_email
+      expect(page).to have_text "Casa Admin was successfully updated."
+
       expect(admin.display_name).to eq expected_display_name
       expect(admin.phone_number).to eq expected_phone_number
     end
   end
+  context "with valid email data" do
```

Comment:
picky: add blank line above

Reply:
Thank you for catching it. I had gone through an earlier PR trying to add that formatting and this one must've slipped by me.